src/app/tests/suites/certification/Test_TC_S_2_2.yaml

```diff
@@ -20,273 +20,692 @@ PICS:
-      disabled: true
-
-    - label:
-          "TH reads attributes SceneCount, CurrentScene, CurrentGroup and
-          SceneValid from DUT."
+      command: "RecallScene"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: 0x0001
+              - name: "SceneID"
+                value: 0x01
+
+    - label: "TH reads attributes SceneValid from DUT."
       PICS: S.S.A0000 && S.S.A0001 && S.S.A0002 && S.S.A0003
-      verification: |
-          DUT provides these attribute values: SceneCount attribute has the value (PIXIT.S.SceneTableEntriesOnFactoryNew + 1) CurrentScene attribute has the value 0x01 CurrentGroup attribute has the value G1 SceneValid attribute has the value TRUE.
-      disabled: true
-
-    - label: "By application specific means, remove the power source from DUT."
-      verification: |
-          DUT is powered off.
-      disabled: true
-
-    - label: "By application specific means, return the power source to DUT."
-      verification: |
-          DUT is powered on.
-      disabled: true
+      command: "readAttribute"
+      attribute: "SceneCount"
+      response:
+          value: 1
+          constraints:
+              type: int8u
+
+    - label: "TH reads attributes CurrentScene from DUT."
+      PICS: S.S.A0000 && S.S.A0001 && S.S.A0002 && S.S.A0003
+      command: "readAttribute"
+      attribute: "CurrentScene"
+      response:
+          value: 0x01
+          constraints:
+              type: int8u
+
+    - label: "TH reads attributes CurrentGroup from DUT."
+      PICS: S.S.A0000 && S.S.A0001 && S.S.A0002 && S.S.A0003
+      command: "readAttribute"
+      attribute: "CurrentGroup"
+      response:
+          value: 0x0001
+          constraints:
+              type: groupid
+
+    - label: "TH reads from the DUT the SceneValid attribute from DUT."
+      PICS: S.S.A0000 && S.S.A0001 && S.S.A0002 && S.S.A0003
+      command: "readAttribute"
+      attribute: "SceneValid"
+      response:
+          value: true
+          constraints:
+              type: uint8
+
+    - label: "Reboot target device"
+      PICS: PICS_SDK_CI_ONLY
+      cluster: "SystemCommands"
+      endpoint: 0
+      command: "Reboot"
+
+    - label: "Reboot target device(DUT)"
+      verification: |
+          Did the DUT successfully reboot?
+      cluster: "LogCommands"
+      command: "UserPrompt"
+      PICS: PICS_SKIP_SAMPLE_APP
+      arguments:
+          values:
+              - name: "message"
+                value: "Please reboot the DUT and enter 'y' after DUT starts"
+              - name: "expectedValue"
+                value: "y"
 
     - label:
           "TH configures AC2 on DUT for all implemented application clusters
           supporting scenes."
-      verification: |
-          DUT is configured with AC2 which is different from AC1.
-      disabled: true
+      cluster: "Level Control"
+      command: "MoveToLevelWithOnOff"
```

Comment:
Again, this does not match the test plan.

Maybe we should do something where we do the level bits if PICS_SDK_CI_ONLY and have a manual step if PICS_SKIP_SAMPLE_APP?